src/coord/src/catalog/builtin.rs

```diff
@@ -1145,6 +1145,18 @@ pub static MZ_CLUSTER_REPLICAS: Lazy<BuiltinTable> = Lazy::new(|| BuiltinTable {
         .with_column("availability_zone", ScalarType::String.nullable(true)),
 });
 
+pub static MZ_AUDIT_LOG: Lazy<BuiltinTable> = Lazy::new(|| BuiltinTable {
+    name: "mz_audit_log",
+    schema: MZ_CATALOG_SCHEMA,
+    desc: RelationDesc::empty()
+        .with_column("uuid", ScalarType::Uuid.nullable(false))
+        .with_column("object", ScalarType::String.nullable(false))
+        .with_column("operation", ScalarType::String.nullable(false))
+        .with_column("op_extra", ScalarType::Jsonb.nullable(false))
```

Comment:
(We generally avoid abbreviations in the system catalog.)

```suggestion
        .with_column("event_details", ScalarType::Jsonb.nullable(false))
```